quinn-proto/src/connection/packet_builder.rs

```diff
@@ -41,6 +41,7 @@ impl PacketBuilder {
         datagram_start: usize,
         ack_eliciting: bool,
         conn: &mut Connection,
+        dst_cid: ConnectionId,
```

Comment:
Style nit: this feels off wrt our usual argument order?

Reply:
Fixed.